Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.messages.in

```diff
@@ -329,10 +329,18 @@ messages -> RemoteGraphicsContextGL NotRefCounted Stream {
     void CreateExternalSync(uint32_t externalSync, WebCore::GraphicsContextGL::ExternalSyncSource arg0) NotStreamEncodable
     void DeleteExternalSync(uint32_t arg0)
     void ClientWaitExternalSyncWithFlush(uint32_t arg0, uint64_t timeout) -> (bool returnValue) Synchronous
+#if ENABLE(WEBXR)
```

Comment:
Here the ifdef support is feature of the "print a message"
We'd need to make the ifdef support for messages be a feature of the "print a list of messages".
So this should be perhaps moved up in `context_msg_webkit_ipc_generator.get_messages`.